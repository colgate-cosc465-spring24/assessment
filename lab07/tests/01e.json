{
    "ases" : [
        {
            "number": 1,
            "prefix": "11.0.0.0/8"
        },
        {
            "number": 2,
            "prefix": "12.0.0.0/8"
        },
        {
            "number": 3,
            "prefix": "13.0.0.0/8"
        },
        {
            "number": 4,
            "prefix": "14.0.0.0/8"
        },
        {
            "number": 5,
            "prefix": "15.0.0.0/8"
        }
    ],
    "relationships" : [
        {
            "customer": 1,
            "provider": 2
        },
        {
            "customer": 2,
            "provider": 3
        },
        {
            "provider": 3,
            "customer": 4
        },
        {
            "provider": 4,
            "customer": 5
        }
    ]
}
