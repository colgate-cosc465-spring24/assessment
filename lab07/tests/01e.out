***** Topology *****
AS 1 (11.0.0.0/8): cust=[]; prov=[2]; peer=[]
AS 2 (12.0.0.0/8): cust=[1]; prov=[3]; peer=[]
AS 3 (13.0.0.0/8): cust=[2,4]; prov=[]; peer=[]
AS 4 (14.0.0.0/8): cust=[5]; prov=[3]; peer=[]
AS 5 (15.0.0.0/8): cust=[]; prov=[4]; peer=[]
***** Routes *****
AS 1
	11.0.0.0/8: 
	12.0.0.0/8: 2
	13.0.0.0/8: 2 -> 3
	14.0.0.0/8: 2 -> 3 -> 4
	15.0.0.0/8: 2 -> 3 -> 4 -> 5
AS 2
	11.0.0.0/8: 1
	12.0.0.0/8: 
	13.0.0.0/8: 3
	14.0.0.0/8: 3 -> 4
	15.0.0.0/8: 3 -> 4 -> 5
AS 3
	11.0.0.0/8: 2 -> 1
	12.0.0.0/8: 2
	13.0.0.0/8: 
	14.0.0.0/8: 4
	15.0.0.0/8: 4 -> 5
AS 4
	11.0.0.0/8: 3 -> 2 -> 1
	12.0.0.0/8: 3 -> 2
	13.0.0.0/8: 3
	14.0.0.0/8: 
	15.0.0.0/8: 5
AS 5
	11.0.0.0/8: 4 -> 3 -> 2 -> 1
	12.0.0.0/8: 4 -> 3 -> 2
	13.0.0.0/8: 4 -> 3
	14.0.0.0/8: 4
	15.0.0.0/8: 
